api/controllers/comment.go

```diff
@@ -78,26 +109,36 @@ func (cc CommentController) UpdateComment(c *gin.Context) {
 		return
 	}
 
-	responses.SuccessJSON(c, http.StatusOK, "Comment Updated Sucessfully")
+	responses.SuccessJSON(c, http.StatusOK, "Comment Updated !!!")
 }
 
 // DeleteComment -> Delete Comment
 func (cc CommentController) DeleteComment(c *gin.Context) {
 	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	userId := c.Query(constants.UID)
 	if err != nil {
 		cc.logger.Zap.Error("Error [DeleteComment] [Conversion Error]: ", err.Error())
 		err := errors.InternalError.Wrap(err, "Failed to Parse Comment ID")
 		responses.HandleError(c, err)
 		return
 	}
+	posts, err := cc.postService.GetOnePost(int64(id), userId)
+	
+	if err != nil {
+		cc.logger.Zap.Error("Error [DeletePosts] [Conversion Error]: ", err.Error())
+		err := errors.InternalError.Wrap(err, "Failed to Parse Posts ID")
+		responses.HandleError(c, err)
+		return
+	}
+	deleteComment := models.Comment{PostId: posts.ID, UserId: userId}
```

Comment:
@Jbholae here we need to check if the comment for the id exists or not, not the post.
```suggestion
	posts, err := cc.commentService.GetOneComment(int64(id), userId)
	
	if err != nil {
		cc.logger.Zap.Error("Error [DeletePosts] [Conversion Error]: ", err.Error())
		err := errors.InternalError.Wrap(err, "Failed to Parse Posts ID")
		responses.HandleError(c, err)
		return
	}
	deleteComment := models.Comment{PostId: posts.ID, UserId: userId}
```

something like this.